### ISSUE TITLE
solve10 always seem to return 0.

### ISSUE THREAD

**Issue by ravin-regmi** [ssldhl/maharishi#1]
9! + 9! < 10! ; 10! + 1! > 10!

**Comment by ssldhl** [comment]
Good catch! I must have missed something. Please update the solution and send a Pull Request.

**Comment by ravin-regmi** [comment]
According to question the program solution seems to be correct. But this question actually does not have solution for integer > 0.
Still waiting for someone to show up with some mathematical solution if exist :smile:
